bin/rector

```diff
@@ -46,6 +52,9 @@ try {
         $changedFilesDetector = $container->get(ChangedFilesDetector::class);
         $changedFilesDetector->setFirstUsedConfig(realpath($rectorConfigsResolver->getFirstResolvedConfig()));
     }
+} catch (PhpVersionException $exception) {
+    print $exception->getMessage() . PHP_EOL;
+    exit(1);
 } catch (Throwable $throwable) {
     $symfonyStyle = (new SymfonyStyleFactory(new PrivatesCaller()))->create();
     $symfonyStyle->error($throwable->getMessage());
```

Comment:
I think the one above can be merged to the `Throwable $throwable` one.

Just the previous:

```php
} catch (Throwable $throwable) {
    $symfonyStyle = (new SymfonyStyleFactory(new PrivatesCaller()))->create();
    $symfonyStyle->error($throwable->getMessage());
}
```